can/simulator/rtos_main.cpp

```diff
@@ -0,0 +1,94 @@
+#include <iostream>
+#include <variant>
+
+#include "FreeRTOS.h"
+#include "can/core/can_message_buffer.hpp"
+#include "can/core/dispatch.hpp"
+#include "can/core/freertos_can_dispatch.hpp"
+#include "can/core/messages.hpp"
+#include "common/core/bit_utils.hpp"
+#include "common/core/freertos_message_buffer.hpp"
+#include "common/core/freertos_task.hpp"
+
+using namespace freertos_message_buffer;
+using namespace can_dispatch;
+using namespace can_message_buffer;
+using namespace can_messages;
+using namespace freertos_can_dispatch;
+using namespace freertos_task;
+
+struct HandlerA {
+    void handle(std::variant<std::monostate, GetSpeedRequest> &m) {
+        std::visit([this](auto o) { this->visit(o); }, m);
+    }
+
+    void visit(std::monostate &m) {
+        std::cout << "HandlerA monostate" << std::endl;
+    }
+
+    void visit(GetSpeedRequest &m) {
+        std::cout << "HandlerA get speed request" << std::endl;
+    }
+};
+
+struct HandlerB {
+    void handle(std::variant<std::monostate, StopRequest> &m) {
+        std::visit([this](auto o) { this->visit(o); }, m);
+    }
+
+    void visit(std::monostate &m) {
+        std::cout << "HandlerB monostate" << std::endl;
+    }
+
+    void visit(StopRequest &m) { std::cout << "HandlerB stop" << std::endl; }
+};
+
+// Handlers for the parsed messages
+static auto a_handler = HandlerA{};
+static auto b_handler = HandlerB{};
+
+// The dispatch targets.
+static auto a = freertos_can_dispatch::FreeRTOSCanDispatcherTarget<
```

Comment:
Build a message buffer that parses GetSpeedRequest messages and sends them to `a_handler`